src/ray/raylet/worker_pool.cc

```diff
@@ -944,7 +953,8 @@ inline WorkerPool::State &WorkerPool::GetStateForLanguage(const Language &langua
 
 inline bool WorkerPool::IsIOWorkerType(const rpc::WorkerType &worker_type) {
   return worker_type == rpc::WorkerType::SPILL_WORKER ||
-         worker_type == rpc::WorkerType::RESTORE_WORKER;
+         worker_type == rpc::WorkerType::RESTORE_WORKER ||
+         worker_type == rpc::WorkerType::GENERAL_IO_WORKER;
```

Comment:
How about calling it UTIL_WORKER? Not sure if there's a better name cc @edoakes

Reply:
PY_WORKER ?